drivers/scsi/iscsi_tcp.c

```diff
@@ -1051,6 +1051,12 @@ static int iscsi_sw_tcp_slave_configure(struct scsi_device *sdev)
 	struct iscsi_session *session = tcp_sw_host->session;
 	struct iscsi_conn *conn = session->leadconn;
 
+	if (conn && conn->persistent_address) {
+		if ((strcmp(conn->persistent_address, "169.254.10.1") == 0) ||
+		    (strcmp(conn->persistent_address, "169.254.10.2") == 0))
```

Comment:
If we don't want to make these IP addresses configurable, can we define ```MACROs``` for them to make them more readable?

Reply:
More readable is always better, but I'd like to make sure I'm following you correctly.  Do you mean that we have something like 
```
#ifdef CONFIG_TRUENAS
#define HA_PRIVATE_CHANNEL_CONTROLLER_A_IP "169.254.10.1"
#define HA_PRIVATE_CHANNEL_CONTROLLER_B_IP "169.254.10.2"
#endif
```
and then use `HA_PRIVATE_CHANNEL_CONTROLLER_A_IP` in the `strcmp`?

If not, could you provide more details?

Reply:
Yes, something like that. Thanks!